komodo-core/src/main/java/org/komodo/repository/RepositoryImpl.java

```diff
@@ -1323,6 +1323,16 @@ public KomodoObject komodoSearches(UnitOfWork transaction) throws KException {
     }
 
     @Override
+    public KomodoObject komodoServersNode(UnitOfWork transaction) throws KException {
+        ArgCheck.isNotNull( transaction, "transaction" ); //$NON-NLS-1$
+        ArgCheck.isTrue( ( transaction.getState() == org.komodo.spi.repository.Repository.UnitOfWork.State.NOT_STARTED ),
+        "transaction state is not NOT_STARTED" ); //$NON-NLS-1$
+
+        komodoEnvironment(transaction);
+        return create(transaction, SERVERS_ROOT, Environment.SERVERS);
```

Comment:
The SERVERS_ROOT grouping node is defined as nt:unstructured in the cnd, so the third arg (tko:servers type) throws an exception when the create is attempted.  Changing the third arg to null will create it without issue.